### PR TITLE
SF-1276: siteParams are available in the store

### PR DESCRIPTION
**Pull request by Vic-Dev**

### CODE REVIEW

File: packages/@storefront/flux-capacitor/src/core/reducers/data/site-params.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Actions from '../../actions';
+import Store from '../../store';
+
+export type Action = Actions.ReceiveSiteParams;
+export type State = Store.SiteParams[];
+
+export const DEFAULT_PARAMS = <any>[];
```
Comment (jmbtutor) [inline packages/@storefront/flux-capacitor/src/core/reducers/data/site-params.ts]: Does this need a type cast? I think an empty array still conforms to the type (maybe you have to annotate the variable with a type).

File: packages/@storefront/flux-capacitor/src/core/store/index.ts [hunk 2]
```diff
@@ -421,6 +424,10 @@ namespace Store {
     }
   }
 
+  export interface SiteParams {
```
[the remark this answers is not in the document]
Comment (Vic-Dev) [reply packages/@storefront/flux-capacitor/src/core/store/index.ts]: I don't want to tie the store interface to the response types. Do you know of one?

[the remark this answers is not in the document]
Comment (jrmykolyn) [reply packages/@storefront/flux-capacitor/src/core/store/index.ts]: It looks like `field` and `value` are pretty common, but I'm not seeing `key`. Withdrawn.

**Comment by jrmykolyn** [comment]
My feedback has been addressed. Approving.

**Pull request merged**